katerina.oblakevich/task_3/src/com/company/Report.java

```diff
@@ -0,0 +1,64 @@
+package com.company;
+
+import java.time.LocalDate;
+import java.util.Random;
+
+public class Report {
+    private int id;
+    private LocalDate date;
+    private String name;
+    private String description;
+    private Status status;
+
+    private final String CHARACT = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+    public Report(int id) {
```

Comment:
It's not correct to create random reports inside constructor. Better to create class like RandomReportGenerator.
Constructor should only initialize fields with default or specified values.